Modal.js

```diff
@@ -0,0 +1,158 @@
+export class Modal {
+
+    #id = 'new.modal'
```

Comment:
`document.querySelector("#new.modal")`:

![giphy](https://github.com/anthonyisensee/incidenttracker/assets/60198052/7076a14d-40dc-425b-868b-af8df074cdbf)

See the following run from the console:

![image](https://github.com/anthonyisensee/incidenttracker/assets/60198052/3b51a226-a7b0-4c6e-8a2b-852434e775eb)

Using periods in IDs is allowed, but [it will cause you additional pain in the long run](https://stackoverflow.com/a/79022/13173288). They are so rarely used that I literally had to double check to make sure they are allowed. I suggest changing to simply `modal`. There's only every supposed to be one of them, after all!

I think this is the final change!!

Reply:
Change has been made.